site/components/pages/Retirements/SingleRetirement/index.tsx

```diff
@@ -62,44 +60,66 @@ export const SingleRetirementPage: NextPage<SingleRetirementPageProps> = ({
 }) => {
   const { asPath, locale } = useRouter();
 
-  const formattedAmount = formatTonnes({
-    amount: retirement.amount,
-    locale: locale || "en",
-  });
+  let formattedAmount: string | undefined;
+  let retiree: string | undefined;
+  let poolTokenName: false | "ubo" | "nbo" | "bct" | "nct" | "mco2" | null;
+  let projectTokenName: "tco2" | "c3t" | null;
+  let carbonTokenName:
+    | "ubo"
+    | "nbo"
+    | "bct"
+    | "nct"
+    | "mco2"
+    | "tco2"
+    | "c3t"
+    | null;
 
-  const retiree =
-    retirement.beneficiary ||
-    props.nameserviceDomain ||
-    concatAddress(props.beneficiaryAddress);
+  let tokenData: {
+    key: string;
+    icon: StaticImageData;
+    label: "UBO" | "NBO" | "BCT" | "NCT" | "MCO2" | "TCO2" | "C3T" | "ICR";
```

Comment:
I feel like we should have some defined types already for these possible strings?

Reply:
this is in `site/` and can be ignored. I'm going to remove this from the PR